scout/server/blueprints/variant/controllers.py

```diff
@@ -372,7 +372,9 @@ def variant(
 
     overlapping_variants, overlapping_outliers = [], []
     if get_overlapping:
-        overlapping_variants, overlapping_outliers = map(list, store.hgnc_overlapping(variant_obj))
+        overlapping_variants, overlapping_outliers = map(
```

Comment:
Right, the outliers: we need to rig `get_omics_variants_hgnc_overlapping()` the same way, don't we?

Reply:
Let's do it in a separate PR, if possible

Reply:
🤷 sure!